diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Chapters.kt

```diff
@@ -0,0 +1,61 @@
+package org.cqfn.diktat.ruleset.constants
+
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.common.config.rules.disabledChapters
+import org.cqfn.diktat.ruleset.utils.isDigits
+import org.jetbrains.kotlin.org.jline.utils.Levenshtein
+
+/**
+ * This class represents the chapters that are in our code style.
+ *
+ * @property number - number of chapter
+ * @property chapterName name of chapter
+ */
+@Suppress("WRONG_DECLARATIONS_ORDER")
+enum class Chapters(val number: String, val chapterName: String) {
```

Comment:
Because in `Warenings` ruleId is String. I can use `number` as Integer, but it's easier to convert a number to a string (as parametrs) than a letter to a number.

Reply:
Ok, let's leave it like this, at least for now.